test/README.md

```diff
@@ -0,0 +1,182 @@
+# Test
+This directory contains tests and testing docs for the knative-sandbox `eventing-kafka` project.
+
+- [Unit Tests](#unit-tests) reside in the codebase alongside the code under test.
+- [Integration Tests](#integration-tests) consist of the `conformance` and `e2e` tests also in this directory.
+- [Performance Tests](#performance-tests) have yet to be completed but will reside here as well.
+
+While some information has been repeated here, it is useful to be familiar with the other
+Knative testing documentation such as...
+- [Serving](https://github.com/knative/serving/blob/master/test/README.md)
+- [Eventing](https://github.com/knative/eventing/tree/master/test/README.md)
+- [Eventing-Contrib](https://github.com/knative/eventing-contrib/blob/master/test/README.md) 
+
+
+## Unit Tests
+The Unit Tests are run by Prow for all Pull Requests.  They are executed by Prow via the [
+pre-submit-tests.sh](./presubmit-tests.sh) script with the `--unit-tests` argument. 
+
+The project [Makefile](../Makefile) contains convenience targets for executing the unit-tests
+for various portions of the project with a code coverage summary...
+
+```
+make test-unit        # All Unit Tests        ( ../pkg/           )
+make test-channel     # Channel Unit Tests    ( ../pkg/channel    )
+make test-common      # Common Unit Tests     ( ../pkg/common     )
+make test-controller  # Controller Unit Tests ( ../pkg/controller )
+make test-dispatcher  # Dispatcher Unit Tests ( ../pkg/dispatcher )
+```
+
+...or you can always run them manually via the `go test` cmd line...
+```
+go test -v pkg/channel/...
+``` 
+
+
+## Integration Tests
+The `End-To-End` and `Conformance` tests are Knative "integration-tests" meant to be run by Prow for
+all Pull Requests.  They are executed by Prow via the [pre-submit-tests.sh](./presubmit-tests.sh)
+script with the `--integration-tests` argument.  The existence of the [e2e-tests.sh](./e2e-tests.sh)
+script will cause them to be run in that use case.
+
+For local development / maintenance of the tests however, it is convenient to be able to run them 
+in various manners which are described here.
+
+>Note - The eventing-kafka project currently has a dependency on the librdkafka C library which
+>requires it to be built in a Docker Container matching the target environment.  Therefore, when
+>developing locally (on MacOS for instance) you will need to use the [local-dev.sh](../hack/local-dev.sh)
+>hack script to start a compliant Docker container / shell when running these commands.
+
```

Comment:
This is no longer true (thankfully!)

Reply:
This comment is from part of the readme that is no longer present